ratis-server/src/main/java/org/apache/ratis/server/impl/RaftServerImpl.java

```diff
@@ -1338,7 +1354,7 @@ private CommitInfoProto updateCommitInfoCache() {
       }
       followerState = updateLastRpcTime(FollowerState.UpdateType.APPEND_START);
 
-      // Check that the append entries are not inconsistent. There are 3
+      // Check that the append ents are not inconsistent. There are 3
```

Comment:
Typo?

Reply:
Sorry for causing this typo , i have fixed it